logging/impl.go

```diff
@@ -20,6 +22,9 @@ type (
 		inUTC bool
 
 		appenders []Appender
+		// Logging to a `testing.T` always includes a filename/line number. We use this helper to
+		// avoid that. This function is a no-op for non-test loggers.
```

Comment:
> avoid that.

Are we just trying to avoid the filename/line number coming from the `logging/impl.go` file? I'm confused what the point of marking as a test helper function is.